doc/meson.build

```diff
@@ -1,6 +1,11 @@
 install_man(['mpd.1', 'mpd.conf.5'])
 
 sphinx = find_program('sphinx-build')
+py = import('python').find_installation()
+if run_command(py, '-c', 'import sphinx_rtd_theme').returncode() != 0
+  error('python module "sphinx_rtd_theme" not found')
```

Comment:
This fails after Sphinx has been detected already. This breaks the whole build for people who are not interested in building documentation, but happen to have Sphinx installed, but this theme is missing.